lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -432,9 +432,16 @@ bool pullCustomize(const PullImportPaths& importedPaths, const UsdMayaPrimUpdate
         const auto&       pulledUfePath = *ufePathIt;
         MFnDependencyNode dgNodeFn(dagPath.node());
 
+        // If the Maya node holds USD type information (e.g. a dummy transform
+        // node which is a stand-in for a non-transform USD prim type), use the
+        // USD type instead.
+        auto              usdTypeNamePlug = dgNodeFn.findPlug("USD_typeName", true);
```

Comment:
This is a crucial improvement to create the proper updater: if USD type information was stored on the Maya node on import / pull (e.g. for dummy transform nodes used for Maya reference prims or scope prims), use the USD type information.